templates/data-providers/index.jsx

```diff
@@ -1,12 +1,11 @@
-import React, { useEffect, useRef, useLayoutEffect } from 'react'
-import { Button } from '@oacore/design'
+import React, { useEffect, useRef, useLayoutEffect, useCallback } from 'react'
```

Comment:
I see you updated React to 17. Are you planning to get rid out of these React import at some point?

Reply:
Yes at some point. There is a bug in nextjs which is preventing us to do it. I cannot find it now but I subscribed to that thread and once it's solved I will do it.